src/components/team/styledTeam.js

```diff
@@ -7,31 +7,62 @@ export const TeamWrapper = styled.section`
     width: 53.61%;  // 772/1440
     margin: auto;
     padding: 0 0 136px;
+
+    @media(max-width: 1439px) {
+        width: 75.39%;
+    }
+
+    @media(max-width: 600px) {
+        padding-bottom: 80px;
+    }
 `
 export const TeamBlocks = styled.ul`
     display: flex;
     justify-content: space-between;
     margin: 80px auto 120.8px auto;
```

Comment:
space between last founder and 'awesome partners' title should be 100px at 375 screen size